tests/integration/issues/github_2295/test_issue.py

```diff
@@ -0,0 +1,59 @@
+import os
+from typing import Iterable, List
+
+import pytest
+
+from jina.drivers.search import KVSearchDriver
+from jina.executors.indexers.keyvalue import BinaryPbIndexer
+from jina.flow import Flow
+from jina import Document, DocumentSet
+
+from tests import validate_callback
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+class SearchDocIndexer(BinaryPbIndexer):
+    def query(self, jina_id: str = None, mongo_ids: List[str] = None):
+        return super().query(jina_id)  # serialized document
+
+    def post_init(self):
+        super().post_init()
+        self.name = 'doc_idx_file'
```

Comment:
maybe make a comment for this to explain why it was necessary and link to the issue that describes the underlying problem

Reply:
good point, done